skins/Chameleon/SkinChameleon.php

```diff
@@ -0,0 +1,555 @@
+            $this->data['namespace_urls'] =
+                array_reverse( $this->data['namespace_urls'] );
+            $this->data['personal_urls'] =
+                array_reverse( $this->data['personal_urls'] );
+        }
+
+        if (strpos($_SERVER["SERVER_NAME"], "stage") !== false) {
+            $this->data['login_url'] = "https://loginstage.microfocus.com/nidp/idff/sso?sid=0";
+            $this->data['signup_url'] = "https://secure-wwwstage.novell.com/selfreg/jsp/createOpenSuseAccount.jsp?login=Sign+up";
+        } else {
+            $this->data['login_url'] = "https://login.microfocus.com/nidp/idff/sso?sid=0";
+            $this->data['signup_url'] = "https://secure-www.novell.com/selfreg/jsp/createOpenSuseAccount.jsp?login=Sign+up";
+        }
+
+        if ($this->data['username']) {
+            $user = User::newFromName( $this->data['username'] );
+            $this->data['gravatar'] = "https://www.gravatar.com/avatar/" . md5( $user->getEmail() );
+        }
+
+        $this->html( 'headelement' );
+?>
+
+<!-- Global Navbar -->
+<nav id="global-navbar" class="navbar navbar-toggleable navbar-inverse bg-inverse">
+
+    <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+    </button>
+
+    <a class="navbar-brand" href="https://www.opensuse.org/">
+        <img src="<?= $wgStylePath ?>/Chameleon/res/images/logo/logo-white.svg" width="48" height="30" class="d-inline-block align-top" alt="openSUSE Logo">
+    </a>
+
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+        <ul class="navbar-nav mr-auto">
+            <li class="nav-item">
+                <a class="nav-link" href="https://software.opensuse.org/"><?= _('Download') ?></a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="https://software.opensuse.org/search"><?= _('Software') ?></a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="https://doc.opensuse.org/"><?= _('Guides') ?></a>
+            </li>
+            <li class="nav-item active">
+                <a class="nav-link" href="/"><?= _('Wiki') ?></a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="https://forums.opensuse.org/"><?= _('Forums') ?></a>
+            </li>
+        </ul>
+    </div>
+</nav>
+
+<!-- Main Wrap -->
+<div id="main-wrap" class="main-wrap d-flex">
+
+    <sidebar class="w-20 hidden-md-down">
+        <div class="container-fluid">
+            <?php $this->renderPortals( $this->data['sidebar'] ); ?>
+            <section>
+                <h4 class="my-3">Sponsors</h4>
+                <?php $arr = array("sponsor_amd.png", 'sponsor_b1-systems.png', 'sponsor_ip-exchange2.png', 'sponsor_heinlein.png'); ?>
```

Comment:
Hardcoding the sponsors here probably isn't the best idea ;-)

I know it's done the same way in the bento theme, so it's not your fault, and it's not urgent - but we should really come up with something that is a bit easier to maintain. 

Ideas:
- have a random_sponsor_logo.php that delivers one of the sponsor logos
- make a $opensuse_sponsors config option (global variable) - adding a sponsor in salt is easier/faster than in the wiki source code